pykern/pkasyncio.py

```diff
@@ -90,6 +90,6 @@ def _init():
         return
     _cfg = pkconfig.init(
         debug=(pkconfig.in_dev_mode(), bool, "enable debugging for asyncio"),
-        server_ip=("127.0.0.1", str, "ip to listen on"),
+        server_ip=("0.0.0.0", str, "ip to listen on"),
```

Comment:
use in_dev_mode to pick the address. In prod, it doesn't make sense to bind to 0.0.0.0.

Reply:
https://github.com/radiasoft/pykern/compare/a82a0b440ff34556a2228f89392ac84ff5b5dbb0..58dbf6d26f4401b1c88a4f90560e16deb4cd1230